exchanges/mock/recording.go

```diff
@@ -311,22 +294,37 @@ const (
 )
 
 // CheckJSON recursively parses json data to retract keywords, quite intensive.
-func CheckJSON(data any, excluded *Exclusion) (any, error) {
-	if d, ok := data.([]any); ok {
+func CheckJSON(data any, excluded *Exclusion, limit int) (any, error) {
+	if value, ok := data.([]any); ok {
 		var sData []any
-		for i := range d {
-			v := d[i]
-			switch v.(type) {
-			case map[string]any, []any:
-				checkedData, err := CheckJSON(v, excluded)
+		for i := range value {
+			switch subvalue := value[i].(type) {
+			case []any:
+				if len(subvalue) > 0 {
+					if _, ok := subvalue[0].(map[string]any); ok && len(subvalue) > limit {
```

Comment:
🚧 I can't see that [this thread](https://github.com/thrasher-corp/gocryptotrader/pull/1968#discussion_r2275295228) has been resolved?